### PR TITLE
Branch view command

### PR DESCRIPTION
**Pull request by wassilililily**
Modifies previous View Company Command to highlight the target company in the GUI.

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -28,6 +28,8 @@ public class ModelManager implements Model {
     private final FilteredList<Job> filteredJobs;
     private final FilteredList<Company> filteredCompanies;
 
+    private Company highlightedCompany;
```
Comment (Quasant) [inline src/main/java/seedu/address/model/ModelManager.java:31]: Could there be a way to get this to be private final?

**Review by Quasant — CHANGES_REQUESTED** [review]
I'm a lil unsure about this implementation. I'll see what Alek has to say about this but I feel like there can definitely be a simpler approach to this.

**Review by AlekKwek18 — DISMISSED** [review]
LGTM

File: src/main/java/seedu/address/model/ModelManager.java [hunk 2]
```diff
@@ -281,9 +283,22 @@ public Predicate<Job> getJobLinkedToCompanyPredicate(Company targetCompany) {
      */
     @Override
     public void showLinkedJobsAndPersonsByCompany(Company targetCompany) {
+        setHighlightedCompany(targetCompany);
         updateFilteredJobList(getJobLinkedToCompanyPredicate(targetCompany));
         updateFilteredPersonList(getPersonLinkedToCompanyPredicate(targetCompany));
-        updateFilteredCompanyList(company -> company.equals(targetCompany));
+        updateFilteredCompanyList(company -> true);
```
[the remark this answers is not in the document]
Comment (wassilililily) [reply src/main/java/seedu/address/model/ModelManager.java:289]: Leaving it out would work also but I just added it for the sake completeness.

File: src/main/java/seedu/address/ui/CompanyListPanel.java [hunk 3]
```diff
@@ -42,6 +49,11 @@ protected void updateItem(Company company, boolean empty) {
                 setText(null);
             } else {
                 setGraphic(new CompanyCard(company, getIndex() + 1).getRoot());
+                if (company.equals(model.getHighlightedCompany())) {
+                    setStyle("-fx-background-color: #A9A9A9;"); //hex code for dark grey
+                } else {
+                    setStyle("");
```
[the remark this answers is not in the document]
Comment (wassilililily) [reply src/main/java/seedu/address/ui/CompanyListPanel.java:55]: It checks which Company is targeted for highlighting in the model manager and changes its background colour

File: src/main/java/seedu/address/model/ModelManager.java (hunk 1 above)
Comment (wassilililily) [reply src/main/java/seedu/address/model/ModelManager.java:31]: I would need to have the constructor for ModelManager to include a highlightedCompany field

File: src/main/java/seedu/address/model/ModelManager.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (Quasant) [reply src/main/java/seedu/address/model/ModelManager.java:289]: I see, this is fine by me then. There is actually a pre-defined predicate that you can use (search for something like unused -> true).

File: src/main/java/seedu/address/ui/CompanyListPanel.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (Quasant) [reply src/main/java/seedu/address/ui/CompanyListPanel.java:55]: Ah as in the else statement. I was wondering if after repeated uses of the view command, I might have multipled highlightedcompanies

**Pull request closed without merging**